R/get_national_data.R

```diff
@@ -44,9 +44,15 @@ get_national_data <- function(country, source = "who", steps = FALSE,
   nation_class$download()
 
   # dataset specifc cleaning
+  if (verbose) {
+    message(paste("- Cleaning", source, "data"))
+  }
   nation_class$clean()
 
   # filter for country of interest
+  if (verbose) {
+    message(paste("- Filtering", source, "data for", country))
```

Comment:
Why do these start with -?

Reply:
Thought it would make them stand out from the info printed by vroom. I'm not attached to them, happy to remove.